projects/baby_driver/scripts/repl_setup.py

```diff
@@ -15,7 +15,10 @@
     init_notifier_gpio_it
 from can_send import can_send_raw, load_dbc, can_send
 
-
+def call_gpio_it():
+    """Initialize gpio intterupt"""
+    init_notifier_gpio_it()
```

Comment:
I added this function so that  gpio_it can be initialized without going through parsing.